topics/metagenomics/tutorials/cami2-assembly/tutorial.md

```diff
@@ -0,0 +1,1085 @@
+
+{%cite noauthor_critical_nodate-1%} (CAMI2) started in 2019. It includes 4 different challenges: 
+
+
+
+* An assembly challenge
+* A genome binning challenge
+* A taxonomic binning challenge
+* A taxonomic profiling challenge
+* Clinical pathogen prediction challenge
+
+Each challenge uses a similar set of benchmark datasets reproducing different environments and different sequencing technologies  long and short reads). The six benchmark datasets—reflecting a range of complexities—were created from 1,680 microbial genomes and 599 circular elements of viruses and plasmid:
+
+
+
+* 2 ‘toy’ datasets created from public data and provided before a challenge, 
+* 3 ‘challenge’ datasets derived exclusively from genomic data that were not publicly available at the time
+    * Marine
+        * Simulated short read and long read shotgun metagenome data from samples at different seafloor locations of a marine environment
+    * Plant-associated
+        * Simulated short read and long read shotgun metagenome data from samples taken from a plant rhizosphere environment
+    * strain madness
+        * Simulated long read and short read shotgun metagenome data, including a large amount of strain-level variation
+* a pathogen detection challenge dataset was offered, based on a clinical metagenome sample of blood from a critically ill patient with hemorrhagic fever of unknown cause.
+
+During the challenges, Datasets for the challenge were available for download only to participants via the [CAMI portal](https://data.cami-challenge.org/). After the challenges, all CAMI benchmark datasets were made available with digital object identifiers (DOIs) ([Table 2](https://www.nature.com/articles/s41596-020-00480-3/tables/2)) and the genomic data are now in public sequence repositories such as the National Center for Biotechnology Information (NCBI) to be used for further benchmarking in the field.
+
+Different tools were run with different parameters on the various datasets. Participants ran their preferred tools with their preferred set of parameters and  submitted their results along with either a Docker container containing the complete workflow, a bioconda script or a software repository with detailed installation instructions, specifying all parameter settings and reference databases used. A total of 5,002 submissions were received from 30 external teams and CAMI developers for the four challenge datasets. Following that, the CAMI developers evaluated the results using standardised metrics and then made sense of the different results described in [Meyer et al, 2020](https://www.biorxiv.org/content/10.1101/2021.07.12.451567v1)
```

Comment:
```suggestion
Different tools were run with different parameters on the various datasets. Participants ran their preferred tools with their preferred set of parameters and  submitted their results along with either a Docker container containing the complete workflow, a bioconda script or a software repository with detailed installation instructions, specifying all parameter settings and reference databases used. A total of 5,002 submissions were received from 30 external teams and CAMI developers for the four challenge datasets. Following that, the CAMI developers evaluated the results using standardised metrics and then made sense of the different results described in [Meyer et al, 2020](https://www.biorxiv.org/content/10.1101/2021.07.12.451567v1)
```

Here the meyer et al would be best replaced with a citation. 

1. Take the doi, and turn it into a bibliography entry via doi2bib.org
2. and then add that to your bib file 
3. replace the meyer et al 2020 with `{% cite ... %}`.

Reply:
ok